src/interfaces/rules-memorandum-interfaces/index.ts

```diff
@@ -0,0 +1 @@
+export * from './rules-memorandum-interface'
```

Comment:
I don't actually like all these sub-folders under src/interfaces. Thoughts?

Reply:
It's better than one folder with all the interfaces in them

Reply:
You think so?

Anyway, we can review this again later. Resolving.